SEED-platform/seed
# 2691 feat/parse meter units

#### Any background context you want to provide?
See ticket.

#### What's this PR do?
Adds conversion factor to imported building sync meter readings.

#### How should this be manually tested?
Upload a building sync file with meter readings, then examine the database and see conversion_factor has been added.

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/2691

#### Screenshots (if appropriate)
A building sync file with a meter reading:
<img width="593" alt="Screen Shot 2021-06-16 at 4 42 31 PM" src="https://user-images.githubusercontent.com/564231/122304363-56319780-cec2-11eb-9583-dece51e57419.png">
The meter reading in the database with a conversion factor:
<img width="886" alt="Screen Shot 2021-06-16 at 4 41 55 PM" src="https://user-images.githubusercontent.com/564231/122304379-5c277880-cec2-11eb-9627-9e3b228e15a9.png">
The conversion factor list showing the correct value has been loaded:
<img width="598" alt="Screen Shot 2021-06-16 at 4 49 21 PM" src="https://user-images.githubusercontent.com/564231/122304672-d5bf6680-cec2-11eb-9a29-7784a2dd6c21.png">
